### PR TITLE
bench NEW=8.19 vs OLD=8.18

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

**Comment by SkySkimmer** [comment]
@coqbot bench

**Comment by SkySkimmer** [comment]
forgot to skip coq tests since they're in the wrong package in 8.18
@coqbot bench

**Comment by SkySkimmer** [comment]
@coqbot bench

File: dev/bench/bench.sh [hunk 1]
```diff
@@ -72,14 +72,16 @@ check_variable () {
 : "${old_coq_opam_archive_git_uri:=https://github.com/coq/opam-coq-archive.git}"
 : "${new_coq_opam_archive_git_branch:=master}"
 : "${old_coq_opam_archive_git_branch:=master}"
+: "${new_coq_version:=8.19+rc1}"
```
Comment (SkySkimmer) [inline dev/bench/bench.sh:75]:
```suggestion
: "${new_coq_version:=dev}"
```
maybe this will work better, lots of packages don't have a 8.19 published yet

**Comment by SkySkimmer** [comment]
@coqbot bench

**Comment by SkySkimmer** [comment]
~~~
│   8.3540   16.9320   8.5780  102.68%   158  coq-category-theory/Instance/Coq/Par.v.html                                                             │
~~~
There is a known slowdown from sort poly (on my machine: 1m08s -> 1m22s, and on that line 11.6s -> 15.5s) , and also a slowdown from #17993 (52s -> 1m07s, and on that line 7.6s -> 12.5s)
cc @herbelin is that PR expect to cause a slowdown?

**Comment by ppedrot** [comment]
I had a quick look, the PR mentioned above introduces costly universe substitutions in `Tacred.compute_fix_wrapper`. In theory we could probably ignore these substitutions.

**Comment by herbelin** [comment]
#17993 fixes an issue with simpl and universe polymorphism by stopping using the cache in the latter case (in `reference_eval`, see test-suite for the bug). Maybe can it be made more efficient by caching cases using the same instance.

**Comment by herbelin** [comment]
In https://github.com/herbelin/github-coq/pull/new/master+pr17993-with-universe-cache, I made the cache dependent on the universe instance (hoping I did not misunderstand EInstance.t). I don't know how to easily test on the given examples though.

**Comment by SkySkimmer** [comment]
see also https://github.com/coq/coq/pull/18492

**Pull request closed without merging**